lib/cdo/slack.rb

```diff
@@ -174,7 +194,7 @@ def self.join_room(name)
   # @param channel_name [String] The name of the Slack channel.
   # @return [nil | String] The Slack channel ID for the channel, nil if not
   #   found.
-  private_class_method def self.get_channel_id(channel_name)
+  def self.get_channel_id(channel_name)
```

Comment:
@wjordan is very clever and realized that I could use this method rather than trying to put channel id in Chef secrets; however, I couldn't find a way to call this method without making it un-private.  Am I doing something incorrectly?

Reply:
I don't know enough about ruby's implementation of private to say for sure, but it definitely seems like the `Slack.get_channel_id` command above should work fine with this method as-is.

That being said, I don't see anything wrong with making this un-private.

Reply:
Excellent!

Aren't you calling this from inside the `Slack` class? Why would you need it outside? Private should be fine, and it is already being used in several places, e.g. https://github.com/code-dot-org/code-dot-org/blob/79a7e21cd4a4f760e3e21d8361ddf58773220403/lib/cdo/slack.rb#L77

Reply:
I think you just need to change https://github.com/code-dot-org/code-dot-org/pull/21150/files#diff-bf2f37bdabbcb29185ddcca470cb4c2dR158
from
```ruby
channel_id = Slack.get_channel_id(channel_name)
```
to
```ruby
channel_id = get_channel_id(channel_name)
```